packages/shared/syncRefs/index.md

```diff
@@ -13,15 +13,24 @@ Keep target refs in sync with a source ref
 import { syncRefs } from '@vueuse/core'
 
 const source = ref('hello')
-const target = ref('target')
+const target1 = ref('target1')
+const target2 = ref('target2')
 
-const stop = syncRefs(source, target)
+const stop = syncRefs(source, [target1, target2])
```

Comment:
I think we should keep the first example simple and easy to understand. We could have a sub title `### Sync with multiple targets for that

Reply:
Thank you for the feedback!
I added another section: 851a300